kwant_to_openfermion.py

```diff
@@ -1,11 +1,61 @@
 import openfermion
 import kwant
+import numpy
+import tinyarray
 
+def _index(lattice_index, spin_index, n_spin):
```

Comment:
Nice, this can be reused if we update OF to allow for a greater range of indices.